t/04.meadow/lsf_detection/ok_lsf91_bw3/lsid

```diff
@@ -4,7 +4,7 @@
 # https://www.bsc.es/support/LSF/9.1.2/lsf_admin/index.htm?cluster_info_viewing_lsf.html~main
 exec cat <<EOF
 IBM Platform LSF Standard 9.1.2, May 5 2013 
-© Copyright IBM Corporation 1992, 2013. 
+(c) Copyright IBM Corporation 1992, 2013.
```

Comment:
Actually, I think I would rather let the non-ASCII characters in, and add an exception in `ascii.t`, so that we can be sure that the LSF meadow detection is compatible with a non-ASCII output.